components/StepsProgress.js

```diff
@@ -202,17 +245,20 @@ const PieHalfCircleRight = styled(PieHalfCircle)`
         `}
 `;
 
-const getBubbleContent = (idx, checked, loading) => {
+const getBubbleContent = (idx, checked, disabled, focused, loading) => {
   if (loading) {
     return <StyledSpinner color={checked ? '#FFFFFF' : 'primary.700'} size={14} />;
   } else if (checked) {
     return <Check color="white" size={14} />;
   }
 
   return (
-    <Span fontWeight={400} fontSize="14px" lineHeight="20px">
-      {idx + 1}
-    </Span>
+    <Circle disabled={disabled} checked={checked} focus={focused}>
+      <circle cx="50%" cy="50%" r="16px"></circle>
+      <text x="50%" y="50%" dominantBaseline="middle" textAnchor="middle">
```

Comment:
I'm not sure why, but with `50%` the text appear a little too high. `51%` seems to resolve the issue.

![image](https://user-images.githubusercontent.com/1556356/105973163-b303f100-608c-11eb-9a4a-49386e79c099.png)

```suggestion
      <text x="50%" y="51%" dominantBaseline="middle" textAnchor="middle">
```